web/src/views/User/UserPage.vue

```diff
@@ -18,6 +18,7 @@ export default defineComponent({
       page: 1,
       itemsPerPage,
     });
+    const search_filter = ref(null);
```

Comment:
Would it be more appropriate to type this as a `Ref<string>` i.e.
```suggestion
    const searchFilter = ref('');
```
(also note the camel case suggestion)

Reply:
Good to know, I was familiar with the python string behavior but not typescript. I'm happy they evaluate the same way.